amical/data_processing.py

```diff
@@ -483,6 +483,8 @@ def select_clean_data(
     verbose=False,
     ihdu=0,
     display=False,
+    show_bad_removed=True,
+    show_nframe=0,
```

Comment:
yeah, that's how I get to bump my contribution number every day :)
```suggestion
    *,
    show_bad_removed=True,
    show_nframe=0,
```